components/SubmitButton/SubmitButton.tsx

```diff
@@ -0,0 +1,16 @@
+import classNames from 'classnames';
+import React, { FunctionComponent } from 'react';
+
+interface BannerAsideProps {
+  classname: string;
```

Comment:
Camel case ... `className` ... extending button element props may take care of that anyway.

Reply:
Updated